src/Bicep.LangServer/Deploy/DeploymentHelper.cs

```diff
@@ -90,30 +90,30 @@ public class DeploymentHelper
                 }
                 catch (Exception e)
                 {
-                    return string.Format(LangServerResources.DeploymentFailedWithExceptionMessage, documentPath, e.Message);
+                    return (LangServerResources.DeploymentFailedResult, string.Format(LangServerResources.DeploymentFailedWithExceptionMessage, documentPath, e.Message));
                 }
             }
 
-            return string.Format(LangServerResources.DeploymentFailedMessage, documentPath);
+            return (LangServerResources.DeploymentFailedResult, string.Format(LangServerResources.DeploymentFailedMessage, documentPath));
         }
 
-        private static string GetDeploymentResultMessage(DeploymentCreateOrUpdateOperation deploymentCreateOrUpdateOperation, string documentPath)
+        private static (string, string) GetDeploymentResultMessage(DeploymentCreateOrUpdateOperation deploymentCreateOrUpdateOperation, string documentPath)
```

Comment:
named tuplet

Reply:
Updated as part of https://github.com/Azure/bicep/commit/c15921c329b811f4103d01048c35655d57d195ea.

Thanks!